papaparse.js

```diff
@@ -1724,7 +1723,7 @@ License: MIT
 			function pushRow(row)
 			{
 				data.push(row);
-				lastCursor = cursor;
+				lastCursor = cursor - inputExpansion;
```

Comment:
Would this pointer math fit better in the [implementation of returnable](https://github.com/stuart-marshall/PapaParse/pull/1/files#diff-fbb8d0b19dc4586fadd80f52a7b49415d2d41a5fdccaeaa728f7fca51cfbbb29L1788)? I'm not sure if there are any other consumers of `lastCursor` inside the `parse` implementation, but it seems like it might be useful to have the value of `lastCursor` always be a valid / correct  index into the string referenced by `input` everywhere within the scope of the `parse` method.

Reply:
Yeah, good suggestion. It works better in returnable.
There aren't any other users of lastCursor - it appears to just be a place to note the value in preparation for returnable. But since there's other math going on in returnable with the cursor it seems reasonable to make the adjustment there.